app/jobs/regular/bulk_invite.rb

```diff
@@ -104,7 +109,16 @@ def send_invite(invite)
             end
           end
         else
-          Invite.invite_by_email(email, @current_user, topic, groups.map(&:id))
+          if @total_invites > Invite::BULK_INVITE_EMAIL_LIMIT
+            invite = Invite.create_invite_by_email(email, @current_user,
+              topic: topic,
+              group_ids: groups.map(&:id),
+              send_email: false,
```

Comment:
Do we still need `send_email` now that we have email status?

Reply:
Yes `send_email` is required here as this argument decides if email should be sent or not, as per [this code](https://github.com/discourse/discourse/blob/e7fe7010b8dcea495a46089c9fd5ec215f3de46e/app/models/invite.rb#L146).

Reply:
Understood, but if we are storing the status of the invite, including a new status for "not required", we could use that instead of a `send_email` parameter. Code that previously passed `send_mail` as true/false could instead use set the correct status.

Reply:
I see. Will update the PR to deprecate `invite.via_email` in favor of `invite.emailed_status`